javascript/packages/fury/lib/meta.ts

```diff
@@ -85,6 +85,12 @@ export const getMeta = (description: TypeDescription, fury: Fury): Meta => {
         needToWriteRef: Boolean(fury.config.refTracking) && false,
         type,
       };
+    case InternalSerializerType.BINARY:
+      return {
+        fixedSize: 8,
+        needToWriteRef: Boolean(fury.config.refTracking) && true,
```

Comment:
```suggestion
        needToWriteRef: Boolean(fury.config.refTracking),
```

seems `&& true` is unnecessary